docs/website/netcdf-java/ncml/index.htm

```diff
@@ -71,7 +71,7 @@ <h3>Using NcML to modify<a name="modify"></a> an existing CDM file </h3>
         write the new file, eg:
         <pre><em><strong></strong>java -Xmx1g -classpath netcdfAll-4.3.jar <strong>ucar.nc2.dataset.NetcdfDataset -in myFile.ncml -out myFile.nc</strong></em></pre>
       </li>
-      <li> (or) In the <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">ToolsUI</a> application, open the NcML file in
+      <li> (or) In the <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">ToolsUI</a> application, open the NcML file in
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.

Reply:
webstart links updated to point to downloads.unidata.ucar.edu.